codebase/sym.py

```diff
@@ -0,0 +1,30 @@
+import math
+from collections import defaultdict
+
+class Sym():
+
+    def __init__(self, c ,s):
+        self.n = 0
+        self.at = c or True
```

Comment:
variable 'self.at' should have a default value of int type. 
```suggestion
        self.at = c or 0
```

Reply:
Nice catch! On the other note, I feel like it is actually the default value for c. So we can pass the default value in args as c=0 rather than setting it inside. This will apply to all such instances from Sym and Num.